### DISCUSSION TITLE
Basic Presence API

### DISCUSSION THREAD

**Discussion by eeews** [isFakeAccount/psnawp#87] [Q&A] [answered]
There's an account called "Spencer" the basic presence API return that the last online date 24/01/2023 on PS5 However when i checked Android PS APP last online was 15/03/2023 Platform: PS APP

The API response
{'basicPresence': {'availability': 'unavailable', 'lastAvailableDate': '2023-01-24T00:57:27.082Z', 'primaryPlatformInfo': {'onlineStatus': 'offline', 'platform': 'PS5', 'lastOnlineDate': '2023-01-24T00:57:27.082Z'}}}

i tested it on my personal account the api returned with Last online was 27/08/2024 Platform: PS4 however i was online on PS APP 
i tested another account called "BBB" Last online was 2023 Platform: PS APP However The Api response returned with this

{'basicPresence': {'availability': 'unavailable', 'primaryPlatformInfo': {'onlineStatus': 'offline'}}}

so why the api only support ps4 and ps5 however on Android PS APP i can see all the the account online on ps4 or ps5 or ps app

**Comment by eeews** [comment]
@isFakeAccount

**Reply by isFakeAccount** [reply]
Both gamertags are private, I couldn't verify the info by myself. Do you have another gamertags whose profile is not private?

**Comment by eeews** [comment]
PSN: fec
PSN: Spencer
PSN: F16
PSN: Friends

These accounts presence Are public And if you checked the PS4 Or PS5 or PS APP you will see that last time online was on ps app however the basic presence API in the library don't return that and return with last time the account was online on PS5 or PS4 Only.

if you cant check these account presence on ps app because you are not friends with them , login with any of your accounts on PS4 or PS5 then logout and then login on PS APP and send request to the basic presence API and you will see that last online was on the console not on PS APP

@isFakeAccount

**Reply by isFakeAccount** [reply, marked as answer]
This will be fixed as part of https://github.com/isFakeAccount/psnawp/issues/99
